resources/views/hardware/view.blade.php

```diff
@@ -281,31 +281,31 @@
                                                         <li> {{ $asset->model->manufacturer->name }}</li>
                                                     @endcan
 
-                                                    @if (($asset->model) && ($asset->model->manufacturer->url))
+                                                    @if (($asset->model->manufacturer) && ($asset->model->manufacturer->url!=''))
                                                         <li>
                                                             <i class="fas fa-globe-americas" aria-hidden="true"></i>
-                                                            <a href="{{ $asset->model->manufacturer->url }}" target="_blank">
-                                                                {{ $asset->model->manufacturer->url }}
-                                                                <i class="fa fa-external-link" aria-hidden="true"></i>
+                                                            <a href="{{ $asset->present()->dynamicUrl($asset->model->manufacturer->url) }}" target="_blank">
+                                                                {{ $asset->present()->dynamicUrl($asset->model->manufacturer->url) }}
+                                                                <i class="fa fa-external-link" aria-hidden="true"><span class="sr-only"></span></i>
```

Comment:
`<span class="sr-only"></span>` should have text in here, as it's meant to indicate what the icon means for screen readers.

Reply:
Sorry about that. I've removed my edits on those lines. I pasted those lines and forgot to remove.